model/Hierarchy.php

```diff
@@ -87,20 +91,22 @@ public function validate(ValidationResult $validationResult) {
 	}
 
 	/**
-	 * Returns the children of this DataObject as an XHTML UL. This will be called recursively on each child,
-	 * so if they have children they will be displayed as a UL inside a LI.
-	 * @param string $attributes Attributes to add to the UL.
-	 * @param string|callable $titleEval PHP code to evaluate to start each child - this should include '<li>'
-	 * @param string $extraArg Extra arguments that will be passed on to children, for if they overload this function.
-	 * @param boolean $limitToMarked Display only marked children.
-	 * @param string $childrenMethod The name of the method used to get children from each object
-	 * @param boolean $rootCall Set to true for this first call, and then to false for calls inside the recursion. You
-	 *                          should not change this.
-	 * @param int $nodeCountThreshold See {@link self::$node_threshold_total}
-	 * @param callable $nodeCountCallback Called with the node count, which gives the callback an opportunity
-	 *                 to intercept the query. Useful e.g. to avoid excessive children listings
-	 *                 (Arguments: $parent, $numChildren)
-	 *                          
+	 * Returns the children of this DataObject as an XHTML UL. This will be called recursively on each child, so if they
+	 * have children they will be displayed as a UL inside a LI.
+	 *
+	 * @param string          $attributes         Attributes to add to the UL
+	 * @param string|callable $titleEval          PHP code to evaluate to start each child - this should include '<li>'
+	 * @param string $extraArg                    Extra arguments that will be passed on to children, for if they
+	 *                                            overload this function
+	 * @param boolean         $limitToMarked      Display only marked children
```

Comment:
We should stick to short-form for these type hints, so:

- int instead of integer (I noticed you've done this below)
- bool instead of boolean